docs/js-client-sdk.iclientconfig.pollaftersuccessfulinitialization.md

```diff
@@ -4,7 +4,7 @@
 
 ## IClientConfig.pollAfterSuccessfulInitialization property
 
-Poll for new configurations (every 30 seconds) after successfully requesting the initial configuration. (default: false)
+Poll for new configurations (every `polingIntervalMs`<!-- -->) after successfully requesting the initial configuration. (default: false)

```

Comment:
There isn't a space here in the source code, so I'm not sure why it autodocs to this.